packages/room-server/README.md

```diff
@@ -49,51 +50,64 @@ yarn install
 ## Running the app
 
 ```bash
+# basic mode
+$ yarn start
 
 # watch mode
-$ yarn start:room-server
+$ yarn start:dev
+
+# debug/watch mode
+$ yarn start:dev
```

Comment:
redundant?

Reply:
mistake, the second one should be `yarn start:debug`